README.md

```
@@ -1,56 +1,55 @@
-<p align="center" >
-  <img src="https://raw.githubusercontent.com/wiki/microblink/blinkid-android/images/logo-microblink.png" alt="Microblink" title="Microblink">
-</p>
+
 
 # BlinkID SDK for Android
 
-The _BlinkID_ Android SDK is a comprehensive solution for implementing secure document scanning and extraction. It offers powerful capabilities for extracting data from a wide range of identification documents.
+The *BlinkID* Android SDK is a comprehensive solution for implementing secure document scanning and extraction. It offers powerful capabilities for extracting data from a wide range of identification documents.
 
+The list of all supported documents and result fields can be found [here](https://docs.microblink.com/blinkid/supported-documents).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)